Add recipe for MPDAF

saimn opened the pull request:
So this is the reason why I needed the fitsio and cfitsio packages ;-)

saimn committed:
Add recipe for MPDAF

saimn commented on recipe_templates/mpdaf/build.sh:
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
```
Not sure what is the best way to be able to use the cfitsio package, this seems fine but maybe there is a better way ?

mwcraig replied:
I'm actually not sure, but if this works, go with it.

saimn replied:
Ok, it works so let's go with it. I pushed a fix to remove the version-dependent url, then it should be fine.

saimn committed:
Fix version-dependent url

The https://pypi.python.org/packages/source/m/mpdaf/mpdaf-{{version}}.tar.gz
url was not working for this package. See also
https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package

mwcraig commented:
Merging, thanks!

saimn commented:
Thanks !
